oe_theme.theme

```diff
@@ -1315,3 +1315,106 @@ function _oe_theme_trim_textfield($text, int $size = 150) {
   // Return the unchanged $text since we don't support other scenarios.
   return $text;
 }
+
+/**
+ * Implements hook_preprocess_paragraph().
+ */
+function oe_theme_preprocess_paragraph__oe_facts_figures(array &$variables): void {
+  $paragraph = $variables['paragraph'];
+  $variables['title'] = $paragraph->get('field_oe_title')->value;
+  $variables['view_all'][0]['path'] = $paragraph->get('field_oe_link')->first()->getUrl();
+  $variables['view_all'][0]['label'] = $paragraph->get('field_oe_link')->first()->get('title')->getValue();
+  $variables['items'] = [];
+
+  foreach ($paragraph->get('field_oe_paragraphs')->referencedEntities() as $sub_paragraph) {
+    /** @var \Drupal\paragraphs\Entity\Paragraph $sub_paragraph */
+    $variables['items'][] = [
+      'icon' => _oe_theme_icons_convert($sub_paragraph->get('field_oe_icon')->value),
+      'value' => $sub_paragraph->get('field_oe_title')->value,
+      'title' => $sub_paragraph->get('field_oe_subtitle')->value,
+      'description' => $sub_paragraph->get('field_oe_plain_text_long')->value,
+    ];
+  }
+}
+
+/**
+ * Helper function to convert icon names.
+ *
+ * @param string $icon_name
+ *   The icon name to be converted.
+ *
+ * @return string
+ *   The converted icon name or digital if the icon name is not supported.
+ */
+function _oe_theme_icons_convert(string $icon_name): string {
```

Comment:
On a second thought let's turn this into a Twig filter, we did the same for file icon. How to do:

In \Drupal\oe_theme_helper\TwigExtension\Filters add:

```php
  public function getFilters(): array {
    return [
      ....
      new \Twig_SimpleFilter('to_icon', [$this, 'toIcon']),
    ];
  }
```

Then convert this function into a method of the `Filters` class and use it directly in the Twig template with `|to_icon`.